src/components/Energiatodistus/ET2018Form.svelte

```diff
@@ -151,6 +152,7 @@
   {schema}
   {inputLanguage}
   {disabled}
+  {disabledRt}
```

Comment:
Itse ehkä antaisin tälle jonkun kuvaavamman isLaskuttaja-tms attribuutin nimen ja syöttäisin sinne suoraan Kayttajat.isLaskuttaja(whoami) tuloksen. Tai sitten avaa tuon disabledRt muotoon disabledRakennustunnus tms. Tämän propdrillaus on imo hieman turhaa kun kaikki vaadittava tuon arvon muodostamiseen passataan jo itsessään tälle tasolle.

Reply:
Viimeisimmässä versiossa tätä ei tässä muodossa ole. Sen sijaan whoami toimitetaan sinne perustietokomponentille, ja se omin päin disabloi mikäli aihetta on.